caiman/utils/visualization.py

```diff
@@ -1153,22 +1153,37 @@ def nb_inspect_correlation_pnr(corr, pnr):
 
         pnr: ndarray
             peak-to-noise image created with caiman.summary_images.correlation_pnr
+
+        cmap: str
+            colormap for plotting corr and pnr images
```

Comment:
Link to what the valid strings are? Or if there's a well-defined term like "a FooBarBaz colormap" use that? This is a little mysterious

Reply:
I'll add link to holoviews colormap which is what this: https://holoviews.org/user_guide/Colormaps.html

Reply:
That's a good solution